### ISSUE TITLE
Misleading console output

### ISSUE THREAD

**Issue by HugoGiraudel** [SassDoc/sassdoc#402]
```sh
» SCSS files successfully parsed.
```

What if the codebase is using `.sass`? This would probably be more generic:

> » Sass files successfully parsed.

**Comment by pascalduez** [comment]
This comes from the stream API, i.e. Gulp where you don't pass in a folder but a glob pattern.
Also when parsing from stdin.

We could rephrase it to something like "Sass sources successfully parsed" or the like.

**Comment by HugoGiraudel** [comment]
Yup.
